config/php-config/php7.2-www.conf

```diff
@@ -187,8 +187,8 @@ pm.max_requests = 100
 ;                          it's always 0 if the process is not in Idle state
 ;                          because memory calculation is done when the request
 ;                          processing has terminated;
-; If the process is in Idle state, then informations are related to the
-; last request the process has served. Otherwise informations are related to
+; If the process is in Idle state, then information are related to the
+; last request the process has served. Otherwise information are related to
```

Comment:
```suggestion
; If the process is in Idle state, then information is related to the
; last request the process has served. Otherwise information is related to
```